pkg/agent/apiserver/apiserver.go

```diff
@@ -131,6 +132,10 @@ func newConfig(npq querier.AgentNetworkPolicyInfoQuerier, bindPort int, enableMe
 	secureServing.BindAddress = net.IPv4zero
 	secureServing.BindPort = bindPort
 
+	// With AntreaProxy enabled, it is possible that K8s Service flows and routes are not ready when apiserver tries to
```

Comment:
I think you mean `proxyAll`, instead of AntreaProxy.

Reply:
Updated.